### PR TITLE
Comments sasha g

### PR DESCRIPTION
**Pull request by aleksandrgolyk**
Created CommentAdvancedView + CommentList + CommentForm (adding commit is working perfectlly fine)
image
image
image

### CODE REVIEW

File: src/app/components/HOCs/ListHOC/ListHOC.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { useEffect, useState } from 'react'
 const ListHOC = (props) => {
   const { children, collectionName, mock } = props
   const [data, setData] = useState(mock)
+  console.log(data)
```
Comment (sashka2131) [inline src/app/components/HOCs/ListHOC/ListHOC.js:9]: Remove all logs

File: src/app/domains/Comment/components/forms/CommentSimpleForm/CommentSimpleForm.template.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+const CommentSimpleForm = (props) => {
+  const { addComment, inputRef } = props
+  const classes = useStyles()
+
+  return (
+    <Container>
+      <Row>
+        <Col>
+          <Box mt={2} mb={2}>
```
Comment (5P4C3GR33D) [inline src/app/domains/Comment/components/forms/CommentSimpleForm/CommentSimpleForm.template.js:19]: Excess wrapper.

File: src/app/domains/Comment/components/forms/CommentSimpleForm/CommentSimpleForm.template.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              </IconButton>
+            </Paper>
+          </Box>
+        </Col>
+      </Row>
+    </Container>
+  )
+}
+
+CommentSimpleForm.propTypes = {}
```
Comment (sashka2131) [inline src/app/domains/Comment/components/forms/CommentSimpleForm/CommentSimpleForm.template.js:45]: add props description and remove default props

File: src/app/domains/Comment/components/forms/CommentSimpleForm/CommentSimpleForm.template.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { ListHOC } from 'app/components/HOCs/ListHOC/index'
+import { CommentAdvancedView } from 'domains/Comment/components/views'
+import { Box, Col, Container, Row } from '@qonsoll/react-design'
+import { useStyles } from './CommentSimpleForm.style'
+import InputBase from '@material-ui/core/InputBase'
+import Divider from '@material-ui/core/Divider'
+import IconButton from '@material-ui/core/IconButton'
+import SendIcon from '@material-ui/icons/Send'
+import { Paper } from '@material-ui/core'
```
Comment (sashka2131) [inline src/app/domains/Comment/components/forms/CommentSimpleForm/CommentSimpleForm.template.js:9]: Fix imports u can compare them into one import, check in other files for example

**Review by 5P4C3GR33D — APPROVED** [review]

**Review by sashka2131 — APPROVED** [review]

**Pull request merged**